# Holds the contact state of left and right feet
std_msgs/Header header
bool left_contact
bool right_contact